src/inc/til.h

```diff
@@ -12,7 +12,6 @@
 #include "til/enumset.h"
 #include "til/pmr.h"
 #include "til/replace.h"
-#include "til/rle.h"

```

Comment:
why not?

Reply:
Initially I did it due to a compilation failure, but then left it because `til::rle` is only used for one class and I felt like reducing the pch compile time like that fits into the theme of the PR.